applications/acdc/src/acdc_agent_util.erl

```diff
@@ -81,7 +81,18 @@ most_recent_db_status(AccountId, AgentId) ->
            ],
     case couch_mgr:get_results(acdc_stats_util:db_name(AccountId), <<"agent_stats/status_log">>, Opts) of
         {'ok', [StatusJObj]} -> {'ok', wh_json:get_value(<<"value">>, StatusJObj)};
-        {'ok', []} -> {'ok', <<"unknown">>};
+        {'ok', []} ->
+        	lager:debug("Could not find a recent status for agent ~p, checking previous modb", [AgentId]),
```

Comment:
This clause should go in a new function; we like to minimize nested clauses and a nicely named function here would be more descriptive that having to parse another nested case clause.

Also, as a matter of style, we don't start log lines with capital letters, and use ~s when printing binaries.